components/TableResult.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div class="px-6 xl:px-20 pb-16">
+    <p class="border-t-[1px] w-full"></p>
```

Comment:
Just use [`border-t`](https://tailwindcss.com/docs/border-width) instead of `border-t-[1px]` because that already sets width to `1px`

Reply:
Thats right!